propel/resource_policy.go

```diff
@@ -1,128 +0,0 @@
-package propel
```

Comment:
We need to keep this, no?

Reply:
yes, I rebased the branch with main, now it is there